ui/raidboss/data/05-shb/dungeon/matoyas_relict.js

```diff
@@ -0,0 +1,203 @@
+import Conditions from '../../../../../resources/conditions.js';
+import NetRegexes from '../../../../../resources/netregexes.js';
+import { Responses } from '../../../../../resources/responses.js';
+import ZoneId from '../../../../../resources/zone_id.js';
+
+export default {
+  zoneId: ZoneId.MatoyasRelict,
+  timelineFile: 'matoyas_relict.txt',
+  triggers: [
+    {
+      id: 'Matoyas Mudman Hard Rock',
+      netRegex: NetRegexes.startsUsing({ id: '547F', source: 'Mudman' }),
+      netRegexDe: NetRegexes.startsUsing({ id: '547F', source: 'Matschmann' }),
+      netRegexFr: NetRegexes.startsUsing({ id: '547F', source: 'tadboue' }),
+      netRegexJa: NetRegexes.startsUsing({ id: '547F', source: 'マッドマン' }),
+      condition: function(data, matches) {
+        return data.me === matches.target || data.role === 'healer';
+      },
```

Comment:
The tankbuster is magical
```suggestion
      condition: Conditions.caresAboutMagical(),
```